### PR TITLE
[[ WidgetUtils ]] Add per-platform placeholder icons to widget utils library

### PR DESCRIPTION
**Pull request by livecodeali**
This patch adds all the platform placeholder icons to the widget utils
library to provide a consistent method of painting platform-specific widget
placeholders when the native content is not available on the current
platform.

This allows removal of some duplicated code in the html5 button widget.

### CODE REVIEW

**Comment by runrevmark** [comment]
@livecode-vulcan review ok efaecc1

**Comment by livecode-vulcan** [comment]
💙 review by @runrevmark ok efaecc18161110491f20a354fe13a4d1dbf904de

File: extensions/modules/widget-utils/widget-utils.lcb [hunk 1]
```diff
@@ -282,10 +282,15 @@ public handler getNativeThemeName() returns String
 	end if
 end handler
 
+constant kWidgetUtilsPlaceholderIcons = {
```
Comment (runrevmark) [inline extensions/modules/widget-utils/widget-utils.lcb]: Travis failed with a syntax error - I think continuation chars are missing here?

Comment (livecodeali) [reply extensions/modules/widget-utils/widget-utils.lcb]: doh!

**Comment by runrevmark** [comment]
@livecode-vulcan review ok 01d93be

**Comment by livecode-vulcan** [comment]
💙 review by @runrevmark ok 01d93be7a24c5acfe7fd524afe451cc47e1a0a06

**Comment by livecode-vulcan** [comment]
😎 test success 01d93be7a24c5acfe7fd524afe451cc47e1a0a06

* try-community-armv6-android-sdk26_ndk16r15: [success](https://vulcan.livecode.com/builders/try-community-armv6-android-sdk26_ndk16r15/builds/494)
* try-community-armv7-android-sdk26_ndk16r15: [success](https://vulcan.livecode.com/builders/try-community-armv7-android-sdk26_ndk16r15/builds/486)
* try-community-arm64-android-sdk26_ndk16r15: [success](https://vulcan.livecode.com/builders/try-community-arm64-android-sdk26_ndk16r15/builds/486)
* try-community-x86-android-sdk26_ndk16r15: [success](https://vulcan.livecode.com/builders/try-community-x86-android-sdk26_ndk16r15/builds/486)
* try-community-x86_64-android-sdk26_ndk16r15: [success](https://vulcan.livecode.com/builders/try-community-x86_64-android-sdk26_ndk16r15/builds/486)
* try-community-js-emscripten-sdk1.35: [success](https://vulcan.livecode.com/builders/try-community-js-emscripten-sdk1.35/builds/2008)
* try-community-universal-ios-iphoneos12.1: [success](https://vulcan.livecode.com/builders/try-community-universal-ios-iphoneos12.1/builds/236)
* try-community-universal-ios-iphonesimulator12.1: [success](https://vulcan.livecode.com/builders/try-community-universal-ios-iphonesimulator12.1/builds/235)
* try-community-universal-mac-macosx10.9: [success](https://vulcan.livecode.com/builders/try-community-universal-mac-macosx10.9/builds/2040)
* try-community-x86-linux-debian8: [success](https://vulcan.livecode.com/builders/try-community-x86-linux-debian8/builds/2030)
* try-community-x86_64-linux-debian8: [success](https://vulcan.livecode.com/builders/try-community-x86_64-linux-debian8/builds/2028)
* try-community-x86-win32: [success](https://vulcan.livecode.com/builders/try-community-x86-win32/builds/2063)
* try-community-x86_64-win32: [success](https://vulcan.livecode.com/builders/try-community-x86_64-win32/builds/2000)

**Pull request merged**